sdk/src/main/java/com/uid2/UID2Manager.kt

```diff
@@ -60,7 +60,7 @@ sealed interface UID2ManagerState {
     data class Established(val identity: UID2Identity) : UID2ManagerState
     data class Refreshed(val identity: UID2Identity) : UID2ManagerState
     object NoIdentity : UID2ManagerState
-    object Expired : UID2ManagerState
+    data class Expired(val identity: UID2Identity) : UID2ManagerState
```

Comment:
I realised that when we "expire", we still have a valid identity that can be returned.